test/docker/kafka/component_integration_test.go

```diff
@@ -0,0 +1,256 @@
+// +build integration
+
+package kafka
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/Shopify/sarama"
+	"github.com/beatlabs/patron"
+	"github.com/beatlabs/patron/component/async"
+	"github.com/beatlabs/patron/component/async/kafka"
+	"github.com/beatlabs/patron/component/async/kafka/group"
+)
+
+func TestKafkaComponent_Success(t *testing.T) {
+	// Timeout the test if it takes too long
+	timeoutTimer := testTimeout(t, 15*time.Second)
```

Comment:
I have left a comment in order to remove thistimeout as `go test` can be provided with a timeout and controlled without the need for a code change.

Reply:
sure thing, I'll remove it.
note that the go test timeout is not test-specific